### PR TITLE
Prep amazon.aws 6.5.4

### PR DESCRIPTION
**Pull request by hakbailey**
Release prep for version 6.5.4

### CODE REVIEW

**Review by GomathiselviS — APPROVED** [review]

**Comment by hakbailey** [comment]
recheck

**Comment by hakbailey** [comment]
recheck

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -2404,11 +2404,23 @@ releases:
...
+      bugfixes:
+      - cloudwatchlogs_log_group_info - Implement exponential backoff when making
+        API calls to prevent throttling exceptions (https://github.com/ansible-collections/amazon.aws/issues/2011).
+      - plugin_utils.inventory - Ensure templated options in lookup plugins are converted (https://github.com/ansible-collections/amazon.aws/issues/1955).
+      release_summary: This release includes bugfixes for the ``cloudwatchlogs_log_group_info``
+        module and the inventory plugins.
+    fragments:
+    - 20240314-cloudwatchlogs_log_group_info-fix-throttling-exceptions.yml
+    - 6.5.4-release-summary.yml
+    release_date: '2024-04-02'
```
Comment (tremble) [inline changelogs/changelog.yaml:2426]:
```suggestion
    release_date: '2024-04-03'
```

**Review by tremble — APPROVED** [review]

**Comment by tremble** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by hakbailey** [comment]
recheck

**Comment by hakbailey** [comment]
recheck

**Comment by hakbailey** [comment]
regate

**Comment by hakbailey** [comment]
regate

**Pull request merged**